components/cards.css

```diff
@@ -0,0 +1,272 @@
+/*---------------------------------------Basic Card------------------------------------------------*/
+
+.card-container{
+    display: flex;
+    gap: 2rem;
+    align-items: center;
+    flex-wrap: wrap;
+}
+
+.card{
+    font-size: 1rem;
+    width: fit-content;
+    height: fit-content;
+    box-shadow: 0 0 5px rgb(0 0 0 / 25%);
+}
+
+.basic-card{
+   
+    width: 15rem;
+    border-radius: 0.25rem;
+}
+.card-content{
+    padding: 0.5rem 1rem;
+}
+.card-header{
+    font-size: 1.125em;
+    font-weight: 600;
+    margin: 0.5rem 0rem;
+    padding: 0rem;
+}
+.card-title, .card-sub-header{
+    color: var(--color-darker-gray);
+    padding: 0rem;
+    margin: 0.75rem 0rem;
+}
+.card-description{
+    margin: 0.5rem 0rem;
+    font-size: 0.9em;
+    color: var(--color-dark-gray);
+}
+.card-btn{
+    margin: 0rem;
+    font-size: 0.8em;
+}
+
+/*---------------------------------------Image Card------------------------------------------------*/
+
+.image-card{
+    width: 20rem;
+    border-radius: 0.25rem;
+}
+.card-image-container{
+    padding: 0rem;
+    margin: 0rem;
+    width: 100%;
+    height: 12rem;
+    border-radius: inherit;
+}
+.card-img{
+    width: 100%;
+    height: 100%;
+    border-radius: inherit;
+}
+.card-button-container{
+    margin: 1rem 0rem;
+    display: flex;
+    gap: 1rem;
+}
+
+/*---------------------------------------Overlay Card------------------------------------------------*/
+
+.overlay-card{
+    margin-top: 1rem;
+    width: 15rem;
+    height: 18rem;
+    overflow: hidden;
+    border-radius: 0.25rem;
+    background-color:var(--color-light-blue);
+    text-align: center;
+    box-shadow: 0 0 5px rgba(0, 0, 0, 0.25);
+    position: relative; 
+}
+
+.overlay-card:hover{
+    box-shadow: 0 0 15px rgba(0, 0, 0, 0.25);
+}
+.overlay-card-image{
+    height: 100%;
+    width: 100%;
+    object-fit:fill;
```

Comment:
Just a suggestion, you should define this for all images in the base.css or reset.css because most images in your code will be kind of same or relative to each other.

Reply:
yes, i'll do that thanks.